Makefile

```diff
@@ -26,7 +26,7 @@ version:
 
 # Cross-compiling
 
-GZ_ARCH     := linux-amd64 linux-i386 linux-armel darwin-i386 darwin-amd64
```

Comment:
I wondered how this worked, and then noticed the `firstword`/`lastword` shenanigans further down. Seems like this'll do what we want

Reply:
Yeah, it's a weird and somewhat arcane Makefile, but it does the job!